lm75bd/lm75bd.c

```diff
@@ -28,7 +28,43 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+  error_code_t errCodeSend;
+  error_code_t errCodeReceive;
+  if (temp == NULL) {
+    return ERR_CODE_INVALID_ARG;
+  } 
+  uint8_t tempWriteBit = 0x00;
+ 
+  errCodeSend = i2cSendTo(devAddr, &tempWriteBit, 1);
+  if (errCodeSend == ERR_CODE_INVALID_ARG) {
+    return ERR_CODE_INVALID_ARG;
+  }
+  uint8_t tempData[2] = {0};
+  errCodeReceive = i2cReceiveFrom(devAddr, tempData, 2);
+  if (errCodeReceive == ERR_CODE_INVALID_ARG) {
+    return ERR_CODE_INVALID_ARG;
```

Comment:
Just check if `errCodeSend != ERR_CODE_SUCCESS` and return the errCode since if the function ever changes and returns some other error code on this call then you won't catch the error here